src/types.ts

```diff
@@ -1,8 +1,10 @@
 export type InfoCard = {
   id: string
-  imgURL?: string
-  title?: string
-  description?: string
+  name?: string
```

Comment:
Is really all of these props is optional?

Reply:
fixed